README.md

```
@@ -1,469 +1,312 @@
+
+    ```bash
+    cd apps/backend && alembic upgrade head
+    ```
+6.  **Start the servers:**
+
+    ```bash
+    ./dev.sh start
+    ```
+
+## usage
+
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)